plugins/scaffolder-backend/api-report.md

```diff
@@ -120,6 +120,7 @@ export function createGithubActionsDispatchAction(options: {
 // @public (undocumented)
 export function createGithubWebhookAction(options: {
   integrations: ScmIntegrationRegistry;
+  webhookSecretDefault?: string;
```

Comment:
sorry for the nitpick - we always name things with "default" as a prefix - can you make these `defaultWebhookSecret`?